core/modules/node/node.pages.inc

```diff
@@ -293,10 +293,47 @@ function node_form($form, &$form_state, Node $node) {
     ),
     '#weight' => -10,
   );
-  $form['options']['status'] = array(
-    '#type' => 'checkbox',
-    '#title' => t('Published'),
-    '#default_value' => $node->status,
+  $is_scheduled = $node_type->settings['status_default'];
+  if ($node->nid) {
+    if ($node->published_on > 0 && $node->status == 0) {
+      $is_scheduled = 2;
+    }
+    elseif ($node->status == 0) {
+      $is_scheduled = 0;
+    }
+    elseif ($node->status == 1) {
+      $is_scheduled = 1;
+    }
+  }
+  $form['options']['publishing_states'] = array(
+    '#type' => 'radios',
+    '#options' => array(
+      1 => t('Publish now'),
+      0 => t('Save draft'),
+      2 => t('Publish later'),
+    ),
+    '#default_value' => $is_scheduled,
+    '#description' => t('Content saved as draft is only accessible to the content creator or site administrators. Content set to Publish later will be saved as draft when the content is created, but will be automatically published on the date and time selected; this requires a <a href="@cron">cron maintenance task</a>.', array('@cron' => url('admin/reports/status')))
```

Comment:
This will need to be split into three different bits of help text and matched with each option. That's a little tricky with FormAPI, let me look up the syntax.... Here we go:
```
$form['publishing_states']['0']['#description'] = t('Will only be accessible to the author or site administrators');
$form['publishing_states']['2']['#description'] =  t('Will be saved as draft now, and automatically published on the date and time selected; this requires a <a href="@cron">cron maintenance task</a>.', array('@cron' => url('admin/reports/status')));
```

We'll also need an access check on `admin/reports/status` before building this link

Reply:
We can probably leave out the cron message here, no? By default, cron runs anyway. And those who know enough to turn it off, probably don't need to be told.

BTW, for the comment automatic closer functionality I don't mention cron.

Reply:
> Let's use author instead of content creator here.

this is the current expression in Backdrop; I didnt add this. (`admin/structure/types/manage/page`). But author is good.
> This will need to be split into different bits of help text to be matched with each option ... FormAPI...

I think this is a bit too busy. With clearer titles for the radios as you suggested earlier, a single simple explanation should be enough. We dont need to hand-hold that much.
> We'll also need an access check 
We can probably leave out the cron message here

I forgot cron runs by default. I think we should delete this